modules/enabling-entra-workload-id-existing-cluster.adoc

```diff
@@ -0,0 +1,246 @@
+// ----
+// $ RELEASE_IMAGE=$(./openshift-install version | awk '/release image/ {print $3}')
+//----
+
+. Extract the list of `CredentialsRequest` objects from the {product-title} release image by running the following command:
++
+[source,terminal]
+----
+$ oc adm release extract \
+  --credentials-requests \
+  --included \
+  --to <path_to_directory_for_credentials_requests> \
+  --registry-config ~/.pull-secret
+----
++
+[NOTE]
+====
+This command might take a few moments to run.
+====
+
+. Set a `AZURE_INSTALL_RG` variable with the {azure-short} resource group name by running the following command:
++
+[source,terminal]
+----
+$ AZURE_INSTALL_RG=`oc get infrastructure cluster -o jsonpath --template '{ .status.platformStatus.azure.resourceGroupName }'`
+----
+
+. Use the `ccoctl` utility to create managed identities for all `CredentialsRequest` objects by running the following command:
++
+[source,terminal]
+----
+$ ccoctl azure create-managed-identities \
+  --name <azure_infra_name> \
+  --output-dir ./output_dir \
+  --region <azure_region> \
+  --subscription-id <azure_subscription_id> \
+  --credentials-requests-dir <path_to_directory_for_credentials_requests> \
+  --issuer-url "${OIDC_ISSUER_URL}" \
+  --dnszone-resource-group-name <azure_dns_zone_resourcegroup_name> \
+  --installation-resource-group-name "${AZURE_INSTALL_RG}"
+----
+
+. Determine whether an Azure pod identity webhook configuration exists on your cluster by running the following command:
++
+[source,terminal]
+----
+$ #need a command here to check this?#
+----
+
+. If there is an existing Azure pod identity webhook configuration, remove it by running the following command:
++
+[source,terminal]
+----
+$ #need a command here to remove this, unless the `oc apply` command below will overwrite it#
+----
+
+. Apply the Azure pod identity webhook configuration for {entra-short} by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f output_dir/manifests/azure-ad-pod-identity-webhook-config.yaml
+----
+
+. Apply the secrets generated by the `ccoctl` utility by running the following command:
++
+[source,terminal]
+----
+$ find ./output_dir/manifests -iname "openshift*yaml" -print0 | xargs -I {} -0 -t oc replace -f {}
+----
+#Are `find` and `xargs` standard Linux commands, or do we need to tell the user to install something to use these? If the latter, is there an OpenShift native (or basic Linux) way to do this?#
++
+This process might take several minutes.
+
+
+. Restart all of the pods in the cluster by running the following command:
++
+[source,terminal]
+----
+$ oc adm reboot-machine-config-pool mcp/worker mcp/master
+----
++
+Restarting a pod updates the `serviceAccountIssuer` field and refreshes the `ServiceAccount` public signing key.
+This process takes a long time.
+#The dev docs version of this has `oc adm reboot-machine-config-pool` followed by both `oc adm wait-[...]` commands used earlier. Does the user need to use both? In what order?#
```

Comment:
This step of rebooting the nodes is necessary to activate the pod identity webhook. The user needs to run the waiting commands to ensure the nodes have completed the reboot and the cluster has reached a stable state.

First, execute `oc adm wait-for-node-reboot nodes --all`. When you see the message "All nodes rebooted", it indicates that the nodes have finished rebooting. 
Then, run `oc adm wait-for-stable-cluster` to confirm that all the cluster operators are in a stable state. If the cluster is stable, it means the cluster is ready for use.